charts/openshift-metering/templates/openshift-reporting/report-queries/pod-memory.yaml

```diff
@@ -365,19 +365,18 @@ spec:
     SELECT
       timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
       timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
-      usage.namespace,
+      usage.namespace AS usage_namespace,
+      request.namespace AS request_namespace,
```

Comment:
We can remove this as the usage and request namespaces are going to be the same.

Reply:
I can't. This is the only thing that prevents the overwrite of the rows from the request ns from being overwritten when the column becomes the usage ns.
Also look at line 382
It can't join without this.
Also it is a value add: now you can see clearly what has usage and what has request.

Reply:
How is it a value add if it's the same namespace at the end of the day? That's just redundant.